framework/src/org/checkerframework/common/wholeprograminference/WholeProgramInferenceScenes.java

```diff
@@ -144,6 +146,51 @@ public void updateInferredConstructorParameterTypes(
     }
 
     /**
+     * Updates the parameter types of the method {@code methodElt} in the Scene
+     * of the method's enclosing class based on the overridden method
+     * {@code overriddenMethod} parameter types.
+     * <p>
+     * For each method parameter in methodElt:
+     *   <ul>
+     *     <li>If the Scene does not contain an annotated type for that
+     *     parameter, then the type of the respective parameter on the
+     *     overridden method will be added to the parameter in the Scene.</li>
+     *     <li>If the Scene previously contained an annotated type for that
+     *     parameter, then its new type will be the LUB between the previous
+     *     type and the type of the respective parameter on the overridden
+     *     method.</li>
+     *   </ul>
+     * <p>
+     * @param methodTree the tree of the method that contains the parameter.
+     * @param methodElt the element of the method.
+     * @param overriddenMethod the overridden method.
```

Comment:
I would say "the AnnotatedExecutableType of the overridden method." Otherwise my first thought is that it is a MethodTree. Same for the Javadoc of updateInferredMethodReceiverType.